packages/application-components/src/components/custom-views/custom-view-loader/messages.ts

```diff
@@ -0,0 +1,11 @@
+import { defineMessages } from 'react-intl';
+
+const messages = defineMessages({
+  loadError: {
```

Comment:
Not sure how to extract this message to the `core` json files in the `i18n` packages.
In that package README [it says](https://github.com/commercetools/merchant-center-application-kit/tree/main/packages/i18n#generating-translation-files) to run the `mc-scripts extract-intl [options]` command, buit it seems `extract-intl` does not exist anymore.
Any help would be much appreciated..

Reply:
Oh! thanks.

Message added here: 25db0c13a6356c58065cb5bb8e3aa937b949b343